eth-connector/src/lib.rs

```diff
@@ -540,6 +595,83 @@ impl Deposit for EthConnectorContract {
     }
 }
 
+#[near_bindgen]
+impl FeeManagement for EthConnectorContract {
+    fn get_deposit_fee_per_silo(&self, silo: Option<AccountId>) -> Option<Fee> {
+        self.fee.deposit_fee_per_silo.get(&silo)
+    }
+
+    fn get_withdraw_fee_per_silo(&self, silo: Option<AccountId>) -> Option<Fee> {
+        self.fee.withdraw_fee_per_silo.get(&silo)
+    }
+
+    fn get_fee_owner(&self) -> AccountId {
+        self.fee
+            .fee_owner
+            .clone()
+            .unwrap_or_else(env::current_account_id)
+    }
+
+    fn calculate_fee_amount(
+        &self,
+        amount: U128,
+        fee_type: FeeType,
+        silo: Option<AccountId>,
+    ) -> U128 {
+        let Some(fee) = (match fee_type {
```

Comment:
Could it be replaced to:
```rust
let Some(fee) = (match fee_type {
    FeeType::Deposit => self.fee.deposit_fee_per_silo.get(&silo),
    FeeType::Withdraw => self.fee.withdraw_fee_per_silo.get(&silo),
}) else { return 0.into() };
```
?

Reply:
No, it cann't, because we need to get the the default fee if the fee for silo isn't set.
  It could be replaced with 
```
Deposit => self.fee.deposit_fee_per_silo.get(&silo).or_else(||self.fee.deposit_fee_per_silo.get(&None)),
Withdraw => self.fee.withdraw_fee_per_silo.get(&silo).or_else(||self.fee.withdraw_fee_per_silo.get(&None)),
```
But the problem of this is that if the `silo` is `None` then read storage will be called twice for the same key `None`.

Reply:
>   because we need to get the default fee if the fee for silo isn't set

Got it.